canonical/ubuntu.com
# [WD-6717] order partner logos according to copy doc in /cloud/public-cloud

## Done

- Order partner logos in /cloud/public-cloud according to the [copy doc](https://docs.google.com/document/d/1ZzUIh9Tfz5SvkrHyo4-izzmUjRka1Tn0UQisKcXGEPA/edit?disco=AAAA7Ee_wf8)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigate to /cloud/public-cloud, scroll down to third section and check the order of the logos of the partners and make sure they are listed in the same order as in the copy doc.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6717

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
